transports/tcp/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 0.41.0
-
+## 0.54.0
```

Comment:
```suggestion
## 0.41.0
```

No need to skip 14 versions :D

Plus, see https://github.com/libp2p/rust-libp2p/blob/master/docs/release.md#development-between-releases for which manifests you need to modify!

Reply:
The milestone describes the version of the `libp2p` crate which is what most people will be using. It re-exports the various subcrates. The project is organised into many individual crates which all have ( / need) their own versions and changelogs.

Reply:
Got it!